a10_octavia/controller/worker/tasks/health_monitor_tasks.py

```diff
@@ -99,12 +103,15 @@ def execute(self, health_mon, vthunder, update_dict):
             url = health_mon.url_path
             expect_code = health_mon.expected_codes
         args = utils.meta(health_mon, 'hm', {})
+        if listeners:
```

Comment:
This isn't a required check. The health monitor can't be created with out the listener being there